Assets/Scripts/GraphArrangerMode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Assets.Scripts
+{
+  public enum GraphArrangerMode
+  {
+    _3D = 0,
+    _2D = 1
```

Comment:
I hate this limitation on enum names but it is what it is :/

Reply:
in this case it doesn't hurt so much